src/utils/functions/notarizePSBT.ts

```diff
@@ -0,0 +1,21 @@
+import { Psbt } from "bitcoinjs-lib";
+import { NotaryAsset } from "../../types/notary-details";
+import getNotarizationSignatures from "./getNotarizationSignatures";
+import syscointx from 'syscointx-js';
+
+/**
+ * Purpose: Notarize Result object from syscointx.createTransaction()/syscointx.createAssetTransaction() if required by the assets in the inputs of the transaction
+ * @param psbt Required. The resulting PSBT object passed in which is assigned from syscointx.createTransaction()/syscointx.createAssetTransaction()
+ * @param notaryAssets Required. Asset objects require notarization, fetch signatures via fetchNotarizationFromEndPoint()
+ * @param rawTx Required. Transaction hex
+ * @returns 
+ */
+export async function notarizePSBT (psbt: Psbt, notaryAssets: Map<string, NotaryAsset>, rawTx: string): Promise<boolean> {
+    const notarizationDone = await getNotarizationSignatures(notaryAssets, rawTx)
+    if (notarizationDone) {
+      return syscointx.addNotarizationSignatures(psbt.version, notaryAssets, psbt.txOutputs)
```

Comment:
Kindly check the return type of `addNotarizationSignatures`.
Should be available on `node_modules/syscointx-js`

Reply:
Added the type `{ output: any; index: number }`, 
found it on https://github.com/syscoin/syscointx-js/blob/master/index.js#L187